frontend/src/metabase/components/Popover.jsx

```diff
@@ -20,12 +21,13 @@ export default class Popover extends Component {
   constructor(props, context) {
     super(props, context);
 
+    this.handleDismissal = this.handleDismissal.bind(this);
```

Comment:
NBD, but I think using public instance fields (`handleDismissal = () => this.`) is more common in the codebase rather than explicit binds.